integrations/access/accessrequest/request_mapping.go

```diff
@@ -0,0 +1,76 @@
+package accessrequest
+
+import (
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/e/lib/expression"
```

Comment:
Won't this break OSS builds?

Reply:
Made a separate PR to move the package to oss [here](https://github.com/gravitational/teleport/pull/38576)
Will update this Pr once that one is in